composer.json

```diff
@@ -14,7 +14,8 @@
     "php": "^7.1",
     "guzzlehttp/guzzle": "~6.0",
     "ext-json": "*",
-    "firebase/php-jwt" : "^5.0"
+    "firebase/php-jwt" : "^5.0",
+    "lcobucci/jwt": "^3.3"
```

Comment:
Why both? Can't `lcobucci/jwt` do everything that `firebase/php-jwt` can do?

Reply:
It can, yes, but the class using the Firebase library is still part of the library. We might be removing in the major but not totally sure yet.